package.json

```diff
@@ -15,22 +15,27 @@
         "dist"
     ],
     "scripts": {
-        "build": "npm-run-all -s \"create-version\" \"build-dev\"",
-        "build-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"build-prod\"",
-        "build-prod": "webpack --mode production",
-        "build-dev": "webpack --mode development",
-        "watch": "npm-run-all -s \"create-version\" \"watch-dev\"",
-        "watch-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"watch-prod\"",
-        "watch-prod": "webpack --watch --mode production",
-        "watch-dev": "webpack --watch --mode development",
+        "build": "npm-run-all -s \"create-version\" \"build-wp-dev\"",
+        "build-prod": "npm-run-all -s \"clean-dist\" \"create-version\" \"build-wp-prod\"",
+        "build-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"build-wp-prod\"",
```

Comment:
Forgot to remove this one?

Reply:
I left the old form as backward compatibility. "build-prod" and "build-sdk" both do the same thing.  JSON doesn't allow comments so I couldn't include a comment to the effect.

Reply:
I think it would be best to remove "build-sdk" - backward compatibility isn't really warranted for this.